README.md

```diff
@@ -2,7 +2,7 @@
 
 Every Nitean creates a repository like this one in their own GitHub account. The reason we do it like this is that the repository might contain personal information and we want every Nitean to be the master of their personal information. If they leave Niteo, they revoke access to their career repo and noone can read it any more. It also gives Niteans clarity into who exactly can read the One on One's notes: only the people listed as collaborators to the repo.
 
-The "meat" of the repo are two issues:
+The gist of the repo are two issues:
```

Comment:
Not an improvement. A gist is a summary. And it's not what I meant. I meant "the main part", but didn't want to use such a dull expression. 

The _bulk_ of the repo, maybe?

Reply:
`There are two issues in the repository:`